version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "3.7.3-SNAPSHOT"
+version in ThisBuild := "3.7.2"
```

Comment:
Whoa! Make sure to keep this as `-SNAPSHOT` since we're not doing a release-build yet.

Reply:
Btw, the latest version is `3.8.1-SNAPSHOT`

Reply:
Yeah I think PR needs to be rebased off of master